atomic_reactor/tasks/binary.py

```diff
@@ -58,6 +58,7 @@ class BinaryPostBuildTask(plugin_based.PluginBasedTask[TaskParams]):
             {"name": "maven_url_sources_metadata"},
             {"name": "verify_media", "required": False},
             {"name": "push_floating_tags"},
+            {"name": "tag_and_push"},
```

Comment:
it should run before group_manifests plugin

Reply:
moved it before `group_manifests` plugin. I am not sure how it'll be able to get all repository annotations :thinking:

Reply:
Could you elaborate on that? I don't think `group_manifests` affects generation of repository annotations, does it?

Reply:
it doesn't I was confused that `post_push_floating_tags` updates repositories.